uaclient/entitlements/livepatch.py

```diff
@@ -24,37 +24,6 @@
 }
 
 
-def configure_snap_proxy(
```

Comment:
I moved the snap functions into its own file originally to solve a  circular import problem. We still have a circular import with importing configure livepatch proxy in config.py, but I ended up adding a couple snap functions as well so I decided to keep the separate snap file. Let me know if you'd rather I move all the snap functions back in here